element-template-generator/core/src/main/java/io/camunda/connector/generator/dsl/CommonProperties.java

```diff
@@ -68,6 +68,7 @@ public static PropertyBuilder retryBackoff() {
     return StringProperty.builder()
         .id("retryBackoff")
         .label("Retry backoff")
+        .feel(FeelMode.disabled)
```

Comment:
Should we generate ETs again in this PR so that all the changes are grouped together?

Reply:
@johnBgood I would do this in a separate PR.